app/workers/fitbit_dump.rb

```diff
@@ -4,17 +4,18 @@ class FitbitDump
    include Sidekiq::Worker
    sidekiq_options :queue => :fitbit, :retry => 5, :unique => true
 
-   def perform(target_address,fitbit_profile_id)
+   def perform(fitbit_profile_id,user_id)
+     @user = User.find_by_id(user_id)
```

Comment:
I don't think you need to load the user here. When you pass it into the mailer, it's going to be serialized automatically (using GlobalID) and loaded again when the mailer is run. Instead, you can just pass along the ID and load the user in the mailer.